conf/evolutions/default/7.sql

```diff
@@ -0,0 +1,10 @@
+# --- !Ups
+
+ALTER TABLE collections ADD COLUMN path_type TEXT;
+UPDATE  collections  SET path_type = 'printSent' where prefill IS NOT NULL;
```

Comment:
Should this be `where prefill IS NULL`? 
Are there tests for this evolution?
Should the `path_type` column be altered to be NOT NULL?